pom.xml

```
@@ -72,6 +72,14 @@
         </repository>
     </repositories>
 
+    <!-- Used for deploying Upgrade Tool to Nexus -->
+    <distributionManagement>
+        <repository>
+            <id>payara-enterprise</id>
```

Please make consistent with main repo
```suggestion
            <id>payara-nexus-enterprise-distributions</id>
```